### PR TITLE
Migrate Tinymce  styles to JS import

### PR DESCRIPTION
**Pull request by belcherj**
#### Changes proposed in this Pull Request

*Migrate Tinymce  styles to JS import

#### Testing instructions

* Create new post
* Observe Tiny MCE styles load

#27515

### CODE REVIEW

**Comment by blowery** [comment]
hmmm

So loading https://hash-746b071b6334cf37a283dedfc6ab7e7c0c41225a.calypso.live/

I see _two_ requests for post-editor.css. One from the section, one from the chunk?

**Comment by blowery** [comment]
ah, it's fine. one is for the section, one for tinymce.

style.css after: 1105270
style.css before: 1159924

saved 54k from style.css before gzip.

**Comment by jsnajdr** [comment]
Two questions that come to mind when reviewing:
1. Does TinyMCE have some own stylesheet? Or is this `style.scss` file all the styling that TinyMCE in Calypso will ever get?
2. Is there any other stylesheet in Calypso that would define or override styles for any TinyMCE elements?

There are many `style.scss` files in the `./plugins` subdirectory, so answer to question 2 is likely to be "yes". Where are these plugin styles loaded? Migrating the main style file and not the plugin ones can change order in which the CSS rules are defined and cause bugs with selector specificity. It's probably necessary to migrate these, too.

**Comment by blowery** [comment]
@belcherj mind if I take over?

**Comment by blowery** [comment]
> Does TinyMCE have some own stylesheet? Or is this style.scss file all the styling that TinyMCE in Calypso will ever get?

TinyMCE has a skin that it pulls in when it spins up. Otherwise it's just our stuff.

> Is there any other stylesheet in Calypso that would define or override styles for any TinyMCE elements?

The only usage of `.tinymce` is in the modified file and in the editor, overrides based on a decendent selector `.post-editor .tinymce` which should be order independent. 

> There are many style.scss files in the ./plugins subdirectory, so answer to question 2 is likely to be "yes". Where are these plugin styles loaded? 

They're all `@import`'d by the tinymce.style included here. Order should be fine.

**Comment by blowery** [comment]
Updated tinymce.js to comply with our linting rules. This now requires a lot more testing.

File: client/components/tinymce/index.jsx [hunk 1]
```diff
@@ -351,7 +354,7 @@ export default class extends React.Component {
 			setup: setup,
 		} );
 
-		autosize( ReactDom.findDOMNode( this.refs.text ) );
+		autosize( this.textInput );
```
Comment (jsnajdr) [inline client/components/tinymce/index.jsx]: `this.textInput.current`?

**Comment by blowery** [comment]
retested with the fixed ref. Toolbar in HTML mode seems good.

**Pull request merged**